src/ssl.c

```diff
@@ -29536,33 +29572,31 @@ void* wolfSSL_GetDhAgreeCtx(WOLFSSL* ssl)
 
         return x509;
     }
+#include "stdio.h"
```

Comment:
Please remove this include for "stdio.h", thanks.

Reply:
Thank you for reviewing.
I removed that line.